Dockerfile

```diff
@@ -125,8 +123,7 @@ FROM rust:1.75-slim-bookworm AS prover-builder
 WORKDIR /workspace
 RUN export DEBIAN_FRONTEND=noninteractive && \
     apt-get update && \
-    apt-get install -y make wget gpg software-properties-common zlib1g-dev libstdc++-12-dev wabt \
-    clang llvm-dev libclang-common-14-dev libpolly-14-dev
+    apt-get install -y make wget gpg software-properties-common zlib1g-dev libstdc++-12-dev wabt
```

Comment:
Updating this to match upstream [here](https://github.com/OffchainLabs/nitro/blob/master/Dockerfile#L123)